fs/xfstests.py

```diff
@@ -118,13 +118,17 @@ def setUp(self):
             self.devices.extend([self.test_dev, self.scratch_dev])
         # mkfs for devices
         if self.devices:
-            line = ('export TEST_DEV=%s' % self.devices[0]).replace('/', '\/')
-            process.system('sed -i "s/export TEST_DEV=.*/%s/g" %s' %
-                           (line, os.path.join(self.teststmpdir, 'local.config')), shell=True)
-            line = ('export SCRATCH_DEV=%s' %
-                    self.devices[1]).replace('/', '\/')
-            process.system('sed -i "s/export SCRATCH_DEV=.*/%s/g" %s' %
-                           (line, os.path.join(self.teststmpdir, 'local.config')), shell=True)
+            l_template = ('export %s=%s').replace('/', '\/')
+            cfg_file = os.path.join(self.teststmpdir, 'local.config')
+            cmd_template = 'sed -i "s/export %s=.*/%s/g" %s'
+            line = (l_template % ('TEST_DEV', self.devices[0])).replace('/', '\/')
+            process.system(cmd_template % ('TEST_DEV', line, cfg_file), shell=True)
+            line = (l_template % ('SCRATCH_DEV', self.devices[1])).replace('/', '\/')
+            process.system(cmd_template % ('SCRATCH_DEV', line, cfg_file), shell=True)
+            line = (l_template % ('TEST_DIR', self.test_mnt)).replace('/', '\/')
+            process.system(cmd_template % ('TEST_DIR', line, cfg_file), shell=True)
+            line = (l_template % ('SCRATCH_MNT', self.scratch_mnt)).replace('/', '\/')
+            process.system(cmd_template % ('SCRATCH_MNT', line, cfg_file), shell=True)
```

Comment:
Wouldn't a file write from python be better ?

Reply:
sure, let me change accordingly